processing/src/main/java/io/druid/query/extraction/BucketExtractionFn.java

```diff
@@ -67,7 +73,8 @@ public String apply(Object value)
   }
 
   @Override
-  public String apply(String value)
+  @Nullable
+  public String apply(@Nullable String value)
   {
     try {
```

Comment:
Null is "expected" here, so suggested to check for it explicitly, rather than operate via exception handling, that is an antipattern in java